chaoscenter/web/src/views/ExperimentDashboardV2/ExperimentDashboardV2Table.tsx

```diff
@@ -208,15 +208,15 @@ const ExperimentDashboardV2Table = ({
               case ExperimentRunStatus.RUNNING:
                 return (
                   <RunExperimentButton
-                    buttonProps={{ disabled: lastExperimentRunStatus === ExperimentRunStatus.RUNNING }}
+                    buttonProps={{ disabled: true }}
```

Comment:
I think we can completely remove this prop in the first place as I don't see it being used. Tagging @amityt / @Saranya-jena to confirm

Reply:
Is this alright @gdsoumya 
```
return (
                  <RunExperimentButton
                    experimentID={data.experimentID}
                    refetchExperiments={refetchExperiments}
                  />
                );
              case ExperimentRunStatus.QUEUED:
                return (
                  <RunExperimentButton
                    experimentID={data.experimentID}
                    refetchExperiments={refetchExperiments}
                  />
                );
```

Reply:
cc: @SahilKr24

Reply:
@arkajyotiMukherjee @gdsoumya @Saranya-jena 
I have shorten it to an if statement. 
PTAL!
```
if (lastExperimentRunStatus === ExperimentRunStatus.RUNNING || lastExperimentRunStatus === ExperimentRunStatus.QUEUED) {
  return (
    <RunExperimentButton
      buttonProps={{ disabled: true }}
      experimentID={data.experimentID}
      refetchExperiments={refetchExperiments}
    />
  );
} else {
  return (
    <RunExperimentButton 
      experimentID={data.experimentID} 
      refetchExperiments={refetchExperiments} 
    />
  );
}
```